lib/ramble/ramble/pipeline.py

```diff
@@ -238,6 +238,12 @@ def _prepare(self):
         super()._prepare()
 
     def _complete(self):
+        # Calculate statistics for repeats and inject into base experiment results
+        for _, app_inst, _ in self._experiment_set.filtered_experiments(self.filters):
+
+            if app_inst.repeats[0] and app_inst.repeats[1] > 0:
```

Comment:
Is there cleaner way to do this? I assume this semantic is essentially `if has_repeats` but it's not super obvious at a glance

Reply:
This is more readable now with the repeats class